includes/definitions.inc.php

```diff
@@ -888,8 +888,8 @@
 $config['os'][$os]['over'][2]['text']  = 'Memory Usage';
 
 $os = 'fabos';
-$config['os'][$os]['text']             = 'Brocade FabricOS';
-$config['os'][$os]['type']             = 'network';
+$config['os'][$os]['text']             = 'FabricOS';
+$config['os'][$os]['type']             = 'storage';
```

Comment:
Indeed, discussions are good.

Me personally, my mind is set and the example I gave in irc is that Cisco MDS fibre channel switches - the same category you are talking about here - are based on NX-OS which is cross platform so you can't then have some devices labelled as storage and some networking. It just doesn't make sense.

I won't be merging this as is, other @librenms/reviewers are more than welcome to feed into the discussion and merge if they feel necessary.